backend/handler/fs.js

```diff
@@ -22,21 +22,18 @@ const fs = require("fs");
 const path = require("path");
 const ResponseCodes = require("../response_codes");
 const systemLogger = require("../logger").systemLogger;
-const farmhash = require("farmhash");
 const utils = require("../utils");
 
-const generateFoldernames = (fileName, dirLevels) => {
+const generateFoldernames = (dirLevels) => {
 	if (dirLevels < 1) {
 		return "";
 	}
 	const folders = [];
-	const minChunkLen = 4;
-	const nameChunkLen = Math.max(fileName.length / dirLevels, minChunkLen);
 
 	for(let i = 0 ; i < dirLevels; i++) {
-		const chunkStart = (i * nameChunkLen) % fileName.length;
-		const fileNameHash = farmhash.fingerprint32(fileName.substr(chunkStart,nameChunkLen) + Math.random());
-		folders.push(fileNameHash & 255);
+		// Generate a relatively large random number for the file directory name
+		const randomNumber = Math.random() * 100000;
```

Comment:
Why do we need a massive number? If we just want it between 0-255 shouldn't we just *255?

Reply:
@carmenfan Yes, 255 should be sufficient. It's only larger to replicate the number that farmhash was previously generating.